### PR TITLE
Add Officers of the Business section

### PR DESCRIPTION
**Pull request by Agent123983**
# What:
- Added Officers of the Business Section.
- This section defines the process and procedure of adding, removing, and responsibilities of an Officer of the Business.
- This section adds the Treasurer and Business Manager as an Officer of the Business

# Why:
- Some reporting submitted to the State of Colorado is required to be submitted by an officer of the organization as defined by the organization's bylaws. This can be an officer of the Board or an officer of the Business. Today, only the Chair, Vice-Chair, or Secretary can submit these reports but as they are fiscal reports, this should be a task done by the current and future treasurers. This change to our bylaws recognizes the Treasurer as an Officer of the organization and allows them to act on behalf of the organization.
- This change will also prepare us for the future, should we grow to a point of needing executive officers such as CEO, CFO, COO... etc. In most cases, executive officers report to the board similarly to how our managers do today but are recognized through organizational bylaws to act on behalf of the organization.

### CODE REVIEW

**Comment by Agent123983** [comment]
@Evan-Robertson please review this new PR draft and modification from the comments of #11

File: bylaws.md [hunk 1]
```diff
@@ -31,15 +31,27 @@
...
+2. Officers shall be elected during a special executive session to be held immediately following a regular election.
+3. Approval shall be accomplished by a majority vote of the Board of Directors.
 4. Elected officers will serve until the next election. Officer vacancies will be filled during the next possible board meeting.
 5. Responsibilities
 	- The Chair shall preside at all Board meetings, appoint committee members, publish the board meeting agendas, and perform other duties as associated with the position.
 	- The Vice-Chair shall assume the duties of the Chair in case of the Chair’s absence or incapacitation. The Vice-Chair shall also preside over committee meetings.
 	- The Secretary shall be responsible for the minutes of the Board, keep all approved minutes in a secure location, and send out copies of minutes to the members.
 
+## Section 2. Officers of the Business
+1. The Officers of the Business shall consist of a Business Manager and Tresurer, appointed by and serving at the pleasure of the Board.
```
Comment (anne-decusatis) [inline bylaws.md]: Tresurer -> Treasurer

Comment (Agent123983) [reply bylaws.md]: Resolved with latest commit, thank you!

**Comment by Agent123983** [comment]
Majority Approval from the Board as of 06/04/2024. Please approve.

**Review by eddiezane — APPROVED** [review]

**Pull request merged**